data-processing/common/parse_equation.py

```diff
@@ -51,7 +51,9 @@ def get_characters(mathml: str) -> List[Character]:
 def get_symbols(mathml: str) -> List[Symbol]:
     soup = BeautifulSoup(mathml, "lxml")
     node_symbols = []
-    for symbol_node in soup.find_all(SYMBOL_TAGS):
+    consecutive_chars: List[Symbol] = []
+    all_indices: List[int] = []
```

Comment:
nit pick: I was a little confused by what all_indices and what consecutive_chars are; can we rename these to something where it's clear that consecutive_chars are symbols (aren't they? I think even things that aren't `<mi>` get added to consecutive_chars?) And can we indicate that `all_indices` are the characters for consecutive characters?

Reply:
consecutive chars can be anything in CHARACTER_TAGS :)